### PR TITLE
Add missing NEWS about td-agent-builder

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+* msi: Fixed a bug that required dll.a was removed unexpectedly.
+  - This bug causes failure to build for C extensions. [GitHub#130]
+* msi: Fixed to use TD_AGENT_TOPDIR in td-agent.conf. [GitHub#131]
+  - It fixes the problem that c:/var is always created automatically.
+    It was changed to respect installed directory (c:/opt/td-agent/var...)
+
+### macOS
+
+* macos: Added to support building macOS installer [GitHub#192]
```
Comment (cosmo0920) [inline NEWS.md:30]: macOS Installer is not included in td-agent 4.0.1 release....

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Td Agent Builder news
+
+## Release v4.0.1 - 2020/08/18
+
+### General
+
+* Unified into `Gemfile` instead of `core_gems.rb` and `plugin_gems.rb`. [GitHub#142]
```
Comment (cosmo0920) [inline NEWS.md:7]: Unified Gemfile is not included in td-agent 4.0.1 release, too.

File: NEWS.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  - It simplify a management of bundled gem files. Note that
+    this change requires newer bundler - bundler 2.2.0.rc1 or later.
+* Added `lockfile:update` rake task to update lock file. [GitHub#161]
+  - Execute `cd td-agent && rake lockfile:update` for updating bundled Gems.
+* Added Serverspec test cases [GitHub#174, #180]
```
Comment (cosmo0920) [inline NEWS.md:12]: ditto.

**Review by cosmo0920 — CHANGES_REQUESTED** [review]

**Comment by kenhys** [comment]
Ah, It should be fixed.

File: NEWS.md [hunk 4]
```diff
@@ -1,17 +1,26 @@
 # Td Agent Builder news
 
-## Release v4.0.1 - 2020/08/18
+## Release v4.0.X -
```
Comment (cosmo0920) [inline NEWS.md:3]: I'm not sure the next release is 4.0.2.
Should we use **Unreleased** or something.

Comment (kenhys) [reply NEWS.md]: I got it. Thanks.

**Review by cosmo0920 — APPROVED** [review]

**Pull request merged**